### ISSUE TITLE
Random vector functional link network: recent developments applications and future directions. (arXiv:2203.11316v1 [cs.NE])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#17093]
Random vector functional link network: recent developments, applications, and future directions. (arXiv:2203.11316v1 [cs.NE])<br>
https://ift.tt/nzHgOD7<br>
<p>Neural networks have been successfully employed in various domain such as classification, regression and clustering, etc. Generally, the back propagation (BP) based iterative approaches are used to train the neural networks, however, it results in the issues of local minima, sensitivity to learning rate and slow convergence. To overcome these issues, randomization based neural networks such as random vector functional link (RVFL) network have been proposed. RVFL model has several characteristics such as fast training speed, simple architecture, and universal approximation capability, that make it a viable randomized neural network. This article presents the comprehensive review of the development of RVFL model, which can serve as the extensive summary for the beginners as well as practitioners. We discuss the shallow RVFL, ensemble RVFL, deep RVFL and ensemble deep RVFL models. The variations, improvements and applications of RVFL models are discussed in detail. Moreover, we discuss the different hyperparameter optimization techniques followed in the literature to improve the generalization performance of the RVFL model. Finally, we give potential future research directions/opportunities that can inspire the researchers to improve the RVFL architecture further.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
